ocamldoc/odoc_test.mli

```diff
@@ -0,0 +1,75 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*             Sebastien Hinderer, projet Cambium, INRIA Paris            *)
+(*                                                                        *)
+(*   Copyright 2022 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+(** Custom generator to perform test on ocamldoc. *)
```

Comment:
This file is only used by the testsuite. Ultimately it should be moved to `testsuite/tests/tool-ocamldoc`. For this PR, it is probably fine to have an empty mli file.